pkg/util/system/socket/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/DataDog/datadog-agent/pkg/util/system/socket
```

Comment:
Wouldn't it be simpler to make a module out of `pkg/util/system` in case you need something else than `socket` ?

Reply:
`pkg/util/system` has dependency on `pkg/util/winutil`, that internally has other dependencies. Hence this approach.

Reply:
That's fair